### PR TITLE
Replace spammy radon image with Wikimedia version

### LINKED ISSUE #4188: Hotlinked image has been changed and now shows advertisement text
## Description

In the "GLM hierarchical model" example notebook, there's supposed to be an image that shows how Radon gas enters a house. This image is hotlinked (rather than embedded or such), and the site hosting the original image has apparently cottoned on to this. They've replaced the image with message saying "visit fix-your-radon" instead.

See for examples:
* https://docs.pymc.io/notebooks/GLM-hierarchical.html
* https://github.com/pymc-devs/pymc3/blob/master/docs/source/notebooks/GLM-hierarchical.ipynb

I'm happy to make a PR to change this image, but I wanted to discuss options first. Either:
* Hotlink to a new image - but accept the risk of this situation arising again
* Use a more reputable image (e.g. [this one from Wikimedia](https://commons.wikimedia.org/wiki/File:CNX_Chem_21_06_RadonExpos.png))
* Store the (binary) image on Git as a binary
* etc...

## Versions and main components

* PyMC3 Version: 3.9.3
* Theano Version: n/a
* Python Version: n/a
* Operating system: n/a
* How did you install PyMC3: n/a

### PR DESCRIPTION
**Pull request by fonnesbeck**
Closes #4188

### CODE REVIEW

File: docs/source/notebooks/GLM-hierarchical.ipynb [hunk 1]
```diff
@@ -776,4 +776,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
+}
```
Comment (MarcoGorelli) [inline docs/source/notebooks/GLM-hierarchical.ipynb]: missing newline's causing a CI failure

**Comment by MarcoGorelli** [comment]
Btw, pre-commit CI has an alpha-preview version out, which should fix pre-commit CI failures for you. You can enable it from https://results.pre-commit.ci/

**Comment by AlexAndorra** [comment]
Ah nice, this looks like exactly what we both talked about a few weeks ago! The owner of the repo would have to activate this option?

**Comment by MarcoGorelli** [comment]
> Ah nice, this looks like exactly what we both talked about a few weeks ago! The owner of the repo would have to activate this option?

If you're a member of pymc3-devs, you can request access, but I think the owner needs to approve

**Pull request merged**